packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
+  .column-right {
+    width: 40%;
+    float: left;
+    border: 1px solid #e0e8ed;
+    color: #012159;
+    label {
+      span {
+        display: block;
+        padding: 10px;
+        margin-bottom: 1px;
+        &:hover {
+          background: #e9f8ff;
+        }
+      }
+      input {
+        display: none;
+        &:checked + span {
+          background: #e9f8ff;
+        }
+      }
+    }
+  }
+  .transfer-buttons {
+    width: 10%;
+    float: left;
+    margin-top: 90px;
+
+    .left, .right {
+      display: block;
+      text-align: center;
+      border: 1px solid #e0e8ed;
+      background-color: rgba(#324E68, .8);
+      color: #fff;
+      padding: 10px;
+      margin: 10px;
+      &:hover {
+        cursor: pointer;
+        background-color: #324E68;
+      }
+    }
+  }
+  ul {
+    margin: 0;
+    list-style: none;
+    padding-left: 0;
+    height: 250px;
```

Comment:
What dictates the `250px`? Is there a way we can make this more generic, so it's not tied directly to a pixel height?

Reply:
Whoops. Sorry this was left in from testing. Height has now been moved to storybook.